packages/authentication/modules/profile/useCurrentProfile/__tests__/useCurrentProfile.test.tsx

```
@@ -0,0 +1,35 @@
+
+jest.mock('@baseapp-frontend/utils', () => ({
+  ...jest.requireActual('@baseapp-frontend/utils'),
+  removeCookie: jest.fn(),
+}))
+
+describe('CurrentProfileProvider', () => {
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it.only('should erase current profile when user logs out', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)